CONTRIBUTING.md

```diff
@@ -23,6 +23,8 @@ inside the `MatrixEventProcessor` class.
 
 * You will need to [setup the bridge](https://github.com/Half-Shot/matrix-appservice-discord/tree/develop#setup-the-bridge) similarly to how we describe,
   but you should setup a homeserver locally on your development machine. We would recommend [Synapse](https://github.com/matrix-org/synapse).
+* We recommend using `yarn` for dependency management (v1 / Classic for the time being).
```

Comment:
TIL that there are newer versions. Are there any known incompatibilities that could bite us/users?

Reply:
Yarn Berry installs in a completely different incompatible layout to Yarn Classic.

Reply:
Anyway, probably be a good idea to enforce node and yarn versions in package.json so users don't get bitten.